packages/python/plotly/plotly/graph_objs/_figurewidget.py

```diff
@@ -15147,7 +15147,7 @@ def add_violin(
             or greater than 4*Q3-3*Q1 are highlighted (see
             `outliercolor`) If "all", all sample points are shown
             If False, only the violins are shown with no sample
-            points and the whiskers extend to the range of the sample.
+            points
```

Comment:
@emmanuelle, I don't think this docstring change should be coming in this PR.  Maybe try rerunning codegen in case you had a different schema around when you generated this.

Reply:
Actually #1988 already removed those. I think they were not added at the right place (I think it was a community PR, I must have merged it without thinking that this should have been in the codegen instead.)

Reply:
What do you suggest to do here @jonmmease ?

Reply:
It looks like this code was somehow generated from the schema for plotly.js 1.51.2 (which this branch shouldn't have yet).  In any case, why don't you try merging in master and then running codegen again.  Hopefully then the diff will only show the colorscale docstring changes.